dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/WorkflowDefinitionMapper.xml

```diff
@@ -189,21 +189,39 @@
         select
         code
         from t_ds_workflow_definition
-        where project_code in
-            <foreach collection="projectCodes" index="index" item="i" open="(" separator="," close=")">
-                #{i}
-            </foreach>
+        <where>
```

Comment:
Please revert this change, this kind of check should in `dao`, we need to avoid using complex logic in mapper.